Sources/Basics/JSON+Extensions.swift

```diff
@@ -89,8 +89,9 @@ extension JSONEncoder {
             } else {
                 encoder.outputFormatting = [.prettyPrinted]
             }
-            #elseif compiler(>=5.3)
-            encoder.outputFormatting = [.sortedKeys, .prettyPrinted, .withoutEscapingSlashes]
+            // `.withoutEscapingSlashes` is not in 5.3 on non-Darwin platforms
+//            #elseif compiler(>=5.3)
+//            encoder.outputFormatting = [.sortedKeys, .prettyPrinted, .withoutEscapingSlashes]
```

Comment:
```suggestion
            // `.withoutEscapingSlashes` is not in 5.3 on non-Darwin platforms
            #elseif compiler(>=5.3) && canImport(Darwin)
            encoder.outputFormatting = [.sortedKeys, .prettyPrinted, .withoutEscapingSlashes]
```

Reply:
Thanks. Looks like `.withoutEscapingSlashes` indeed works ok so I undid this change. https://github.com/apple/swift-package-manager/pull/3097/commits/ec24461f08e9684df36f2abe415d90978ffe169f